src/features/chains/types.ts

```diff
@@ -2,10 +2,31 @@ import { z } from 'zod';
 
 import { ChainMetadata, ChainMetadataSchema } from '@hyperlane-xyz/sdk';
 
+export enum ProtocolType {
+  Ethereum = 'ethereum',
+  Sealevel = 'sealevel',
+}
+
+export const ProtocolSmallestUnit = {
+  [ProtocolType.Ethereum]: 'wei',
+  [ProtocolType.Sealevel]: 'lamports',
+};
+
 export const ChainMetadataExtensionSchema = z.object({
-  logoURI: z.string().optional(),
+  // Extended chain metadata for multi-env support
+  // TODO move to SDK (see https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/2203)
```

Comment:
Is this basically https://github.com/hyperlane-xyz/hyperlane-monorepo/issues/2214?

Reply:
Yes!